draft-ietf-jsonpath-base.md

```diff
@@ -1400,25 +1400,24 @@ Note that `..` on its own is not a valid selector.
 
 A descendant selector selects zero or more descendants of a node.
 
-A nodelist enumerating the descendants is known as a _descendant nodelist_ when:
+The resultant nodelist of a descendant selector is the result of applying a selector,
+depending on the variant of the descendant selector, to a
+nodelist enumerating the node and its descendants such that:
 
 * nodes of any array appear in array order,
 * nodes appear immediately before all their descendants.
 
 This definition does not stipulate the order in which the children of an object appear, since
 JSON objects are unordered.
 
-
-The resultant nodelist of a descendant selector is the result of applying a selector
-(or no selector), depending on the variant of the descendant selector, to a
-descendant nodelist, as shown below:
+The selector to apply is shown below:
 
 | Variant | Selector to apply | Comment |
 | :---: | :---: | ------- |
 | `..<name>` | `.<name>`| [Dot selector](#dot-selector)  |
-| `..*` | _none_ | All descendants |
+| `..*` | `.*` | All descendants |
```

Comment:
This is an interesting change and brings up and interesting point.

`.*` doesn't select the current node, only immediate children.  Now I'm questioning whether `..*` should select the current node.  I'll need to think about this some.

Reply:
The comparison project shows a consensus for `..*` _not_ including the current node. See [this test](https://cburgmer.github.io/json-path-comparison/results/dot_notation_with_wildcard_after_recursive_descent.html).

Reply:
I think only including children is the right way.  I can't think of an example where it'd be necessary to include the current node, and it has similarity with `.*` this way.